src/style.css

```diff
@@ -1,3 +1,71 @@
 body {
   margin: 0;
+  padding: 0;
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+}
+
+@media (max-width: 1024px) {
+  body {
+    max-width: 1024px;
+  }
+}
+
+@media (min-width: 1025px) and (max-width: 1200px) {
+  body {
+    max-width: 1200px;
+  }
+}
+
+.header {
+  height: 60px;
```

Comment:
![Uploading image.png…]()

Reply:
Sorry, I can’t  see changes here. It just says "uploading image.png..." but no img

Reply:
let's skip this comment